### PR TITLE
Actions run tests

### PR DESCRIPTION
**Pull request by chriswedgwood**
Fixing broken CI with addition of postgres service

### CODE REVIEW

File: .github/workflows/pythonpackage.yml [hunk 1]
```diff
@@ -4,20 +4,27 @@ on: [push]
...
     runs-on: ubuntu-latest
     strategy:
       max-parallel: 4
       matrix:
         python-version: [3.7]
-
     services:
       postgres:
-        image: postgres:11
+        image: postgres:10.8
```
Comment (tintinnabulate) [inline .github/workflows/pythonpackage.yml:16]: Any reason for downgrading Postgres image ?

File: .github/workflows/pythonpackage.yml [hunk 2]
```diff
@@ -34,9 +41,11 @@ jobs:
...
    #     flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+  
+    - name: Run migrations
+      run: python manage.py migrate
     - name: Test with pytest
       run: |
         pip install pytest
         pip install pytest-django
-        pytest -h
         pytest
```
Comment (tintinnabulate) [inline .github/workflows/pythonpackage.yml:51]: This reminds me... We are using pytest in here in CI, but it's not in requirements.txt . I think it should be in requirements.txt.

File: .github/workflows/pythonpackage.yml (hunk 1 above)
Comment (chriswedgwood) [reply .github/workflows/pythonpackage.yml]: Upgraded 👍

File: .github/workflows/pythonpackage.yml (hunk 2 above)
Comment (chriswedgwood) [reply .github/workflows/pythonpackage.yml]: added to requirements. We should probably think about splitting requirements into dev/test/prod etc?

**Pull request merged**